src/scsi.c

```diff
@@ -137,7 +137,7 @@ static void dump_p10(unsigned char *p)
 	int i;
 	printf("----------------------------------------------\n");
 	for (i = 0; i < 8; i++, p += 16) {
-		printf("%p: %02x %02x %02x %02x %02x %02x %02x " \
+		printf("%s: %02x %02x %02x %02x %02x %02x %02x " \
 		       "%02x %02x %02x %02x %02x %02x %02x %02x %02x\n", p,
```

Comment:
Please don't print raw binary data as a string. Either cast p to void * to silence the warning or just remove it from printf.

Reply:
Changed to (void *) and the %s is again %p.